models.py

```diff
@@ -27,10 +28,27 @@ def transform(self, datagen, validation_datagen=None):
             outputs[name] = softmax(prediction, axis=1)
         return outputs
 
+    def set_model(self):
+        encoder = self.architecture_config['model_params']['encoder']
+        if encoder == 'VGG11':
+            self.model = UNet11(num_classes=2, pretrained=True)
+            self._initialize_model_weights = lambda: None
```

Comment:
This handling of no model's weights initialization seems pretty ugly, but I didn't know any better.

Reply:
@taraspiotr Yeah that is why the idea before was to override this method. As of now I don't see a better option either